src/Page/Form.purs

```diff
@@ -204,6 +211,27 @@ component =
         , HP.id_ field.key
         , HE.onChecked (HE.input $ UpdateKey field.key <<< UserInput <<< Lynx.Data.Expr.Boolean)
         ]
+    Typeahead typeahead ->
```

Comment:
Hmm, we might want to account for the fact that eventually we'll need to support two different typeaheads: single and multi (`Maybe item` vs `Array item`)

Perhaps we should name this `TypeaheadSingle`